### DISCUSSION TITLE
WebSocket connection failed

### DISCUSSION THREAD

**Discussion by KamillaPup** [Difegue/LANraragi#881] [Q&A]
I redirect port 3000 to a subdomain (port 443) using a proxy a LANraragi docker container in Plesk. I am using LetsEncrypt.

Whenever I do a batch task, even apply tag rules, it fails with the below message.
```
WebSocket connection to 'wss://sub.domain.tld/batch/socket' failed: 
Batch.startBatch @ batch.js?0.8.90:170
```
Let me know some things I can try.

Thanks!

**Comment by Difegue** [comment]
websocket runs under the same port as http/https normally - you might want to check the browser console for more information.

**Reply by KamillaPup** [reply]
That is from the console. I'm using Plesks built in docker so in docker I have LRRs port mapped from 3000 internal to 3024 external and then I have a subdomain in Plesk proxy 3024 to serve it on 443. Even batch tagging gives me the error 1006.

If I run LRR from domain.tld:3024, no problems.

Side question: why does filename parser tag parody series as "series" and e-h plugin uses "parody"? Basically I just want only the romaji title in the title field. I thought there was a way to have the E-H plugin do it by clicking save archive title but that doesn't work.

**Reply by Difegue** [reply]
e-h uses parody because that's what they use over there, while the LRR default is series. Both namespaces are accepted so that probably shouldn't cause much trouble. You could probably get the romaji title with save archive title but that only really uses the title from the remote EH entry, so if it's still in japanese you're outta luck  

As for the WS issue, since you're proxying to use SSL, you might need to make sure the matching `Upgrade` headers are added so that it swaps to WSS properly -- See https://github.com/Difegue/LANraragi/pull/149 and https://github.com/Difegue/LANraragi/commit/1b8326714ca04987bcf98ed37dd7e7bfcd8f1b0b
